doc/source/performance/xtradb_performance_improvements_for_io-bound_highly-concurrent_workloads.rst

```diff
@@ -158,30 +156,53 @@ compiled with ``UNIV_PERF_DEBUG`` C preprocessor define.
    :dyn: Yes
    :vartype: Boolean
 
+.. variable:: innodb_flush_method
```

Comment:
This is not the canonical reference of this variable, which is fully upstream one. This section should only describe what does the parallel doublewrite buffer do for the different settings, same way it is in 5.7 docs.